diff --git a/node_modules/react-material-workspace-layout/HeaderButton/index.js b/node_modules/react-material-workspace-layout/HeaderButton/index.js
index 0058c2b..7587291 100644
--- a/node_modules/react-material-workspace-layout/HeaderButton/index.js
+++ b/node_modules/react-material-workspace-layout/HeaderButton/index.js
@@ -32,11 +32,9 @@ var ButtonInnerContent = styled("div")(function (_ref2) {
   };
 });
 var IconContainer = styled("div")(function (_ref3) {
-  var textHidden = _ref3.textHidden;
+  var theme = _ref3.theme;
   return {
     color: colors.grey[700],
-    height: textHidden ? 32 : 20,
-    paddingTop: textHidden ? 8 : 0,
     "& .MuiSvgIcon-root": {
       width: 18,
       height: 18
@@ -69,7 +67,13 @@ export var HeaderButton = function HeaderButton(_ref5) {
     onClick: onClick,
     disabled: disabled
   }, /*#__PURE__*/React.createElement(ButtonInnerContent, null, /*#__PURE__*/React.createElement(IconContainer, {
-    textHidden: hideText
+    style: hideText ? {
+      height: 32,
+      paddingTop: 8
+    } : {
+      height: 20,
+      paddingTop: 0
+    }
   }, icon || getIcon(name, customIconMapping)), !hideText && /*#__PURE__*/React.createElement(Text, null, /*#__PURE__*/React.createElement("div", null, name)))));
 };
 export default HeaderButton;
\ No newline at end of file
